app/src/main/java/se/kth/sda/othello/imp/BoardImp.java

```diff
@@ -12,13 +13,21 @@
 public class BoardImp implements Board {
     Node nodes[][] = new Node[8][8];
 
+    ArrayList<Node> initialNodes = new ArrayList<>();
+
     // TODO: Correctly initialize the four inital nodes
     public BoardImp() {
         for (int i=0; i<8; i++) {
             for (int j=0; j<8; j++) {
                 nodes[i][j] = new NodeImp(i,j);
             }
         }
+        initialNodes.add(new NodeImp(3,3));
```

Comment:
Still feels like hard coding. May be let's add flexible size for the board so we will have flexible initial nodes as well.

Reply:
Agree. But the size of the board is also hardcoded. 
We can get rid of exact numbers when we start working on another task and that is not in the scope of the current sprint.